# Frame hessian is the internal structure used in dso
float64 time_stamp # timestamp
int32 frame_id    # key-frame ID

